app/models/concerns/twitch_user.rb

```diff
@@ -4,50 +4,37 @@
 module TwitchUser
   extend ActiveSupport::Concern
 
-  included do
-    def sync_twitch_follows!
-      success = false
-      ActiveRecord::Base.transaction do
-        current_followed_users = User.where(twitch_id: Twitch::Follows.followed_ids(twitch_id))
-        old_followed_users = twitch_followed_users
-
-        TwitchUserFollow.import((current_followed_users - old_followed_users).map do |u|
-          TwitchUserFollow.new(from_user: self, to_user: u)
-        end)
-
-        (old_followed_users - current_followed_users).each do |u|
-          TwitchUserFollow.find_by(from_user: self, to_user: u).destroy
-        end
-
-        success = true
-      end
-      update(twitch_user_follows_checked_at: nil) unless success
-    end
-
-    def twitch_sync!
-      body = Twitch::User.get(twitch_id)
+  def sync_twitch_follows!
+    ActiveRecord::Base.transaction do
+      current_followed_users = User.where(twitch_id: Twitch::Follows.followed_ids(twitch_id))
 
-      basic_info = JSON.parse(body)
+      TwitchUserFollow.where(from_user: self, to_user: (twitch_followed_users - current_followed_users)).delete_all
```

Comment:
Not a problem, just a side note that `delete_all` doesn't run any callbacks associated with the model, if we ever end up adding any to this particular model.  Same as the note about `delete_all` that I put on the segment history and run history models.